spec/presenters/result_set_presenter_spec.rb

```diff
@@ -335,6 +335,94 @@
     end
   end
 
+  describe "#top_results?" do
```

Comment:
There isn't a method in ResultSetPresenter called `top_results?` so this should be renamed.